flake8diff/flake8.py

```diff
@@ -147,6 +152,22 @@ def get_color_kwargs(self, details):
             header=self.color_getter('header')(get('header')),
         )
 
+    def should_include_violation(self, violation, changed_lines):
+        """
+        Check if given violation should be included.
+
+        This method accounts for strictness modes.
+        """
+        strict_mode = self.options.get('strict_mode', 'only_lines')
+        if strict_mode == 'only_lines':
+            return violation['line_number'] in changed_lines
+        elif strict_mode == 'file':
+            return True
```

Comment:
the idea was that in case in the future we will need strict modes which will account individual violations, the function will have access to it. since we dont need that know its a good point to move this outside of the loop. thanks